packages/rum-core/src/domain/assembly.ts

```diff
@@ -134,13 +129,13 @@ export function startRumAssembly(
     LifeCycleEventType.RAW_RUM_EVENT_COLLECTED,
     ({ startTime, rawRumEvent, domainContext, savedCommonContext, customerContext }) => {
       const viewHistoryEntry = viewHistory.findView(startTime)
-      const urlContext = urlContexts.findUrl(startTime)
       const session = sessionManager.findTrackedSession(startTime)
-      if (session && viewHistoryEntry && urlContext) {
+
+      if (session && viewHistoryEntry) {
         const commonContext = savedCommonContext || getCommonContext()
         const actionId = actionContexts.findActionId(startTime)
 
-        const rumContext: RumContext = {
+        const rumContext: Partial<CommonProperties> = {
```

Comment:
Change unrelated to the PR, but we can remove `RumContext` and directly use the `CommonProperties` type generated by the rum-event-format